Alignment/OfflineValidation/bin/Zmumumerge.cc

```diff
@@ -0,0 +1,310 @@
+#include <iostream>
+#include <TString.h>
+#include <TCanvas.h>
+#include "TBranch.h"
+#include "TTree.h"
+#include "TSystem.h"
+#include "TH1F.h"
+#include "TH1D.h"
+#include "TF1.h"
+#include "TMath.h"
+#include "TLegend.h"
+#include "TLatex.h"
+#include <TH2F.h>
+#include <TCut.h>
+#include <TStyle.h>
+
+#include <TChain.h>
+#include <TFile.h>
+#include <cmath>
+#include <string>
+
+#include "TH1.h"
+#include "TROOT.h"
+#include "TPad.h"
+#include "TRandom.h"
+#include "THStack.h"
+#include "TH2.h"
+#include "TLatex.h"
+#include "TPaveText.h"
+#include "TFrame.h"
+#include "TAxis.h"
+#include <iostream>
+#include <fstream>
+#include <iomanip>
+#include <TCut.h>
+
+#include <RooFit.h>
+#include <RooRealVar.h>
+#include <RooDataHist.h>
+#include <RooDataSet.h>
+#include <RooHistPdf.h>
+#include <RooPlot.h>
+#include <RooAddPdf.h>
+//#include <RooGlobalFunc.h>
+#include <RooGaussian.h>
+#include <RooPolynomial.h>
+#include <RooGenericPdf.h>
+#include <RooFormulaVar.h>
+#include <RooChebychev.h>
+#include <RooExponential.h>
+#include <RooFitResult.h>
+#include <RooWorkspace.h>
+#include <RooMCStudy.h>
+#include "RooHist.h"
+#include "RooConstVar.h"
+#include "RooMsgService.h"
+#include "RooVoigtian.h"
+#include "FitWithRooFit.cc"
+#include "toolbox.h"
+#include "Options.h"
+#include <boost/filesystem.hpp>
+#include <boost/property_tree/ptree.hpp>
+#include <boost/property_tree/json_parser.hpp>
+#include <boost/optional.hpp>
+
+using namespace RooFit;
+using namespace std;
+using namespace AllInOneConfig;
+namespace pt = boost::property_tree;
+static const int colors_array[] = {4,  8,  2,  3,  4,  7,  30, 6,  9,  46, 36, 8,  10, 11,
+                                   12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24};
+/*--------------------------------------------------------------------*/
+void makeNicePlotStyle(RooPlot* plot) {
+  plot->GetXaxis()->CenterTitle(true);
+  plot->GetYaxis()->CenterTitle(true);
+  plot->GetXaxis()->SetTitleFont(42);
+  plot->GetYaxis()->SetTitleFont(42);
+  plot->GetXaxis()->SetTitleSize(0.05);
+  plot->GetYaxis()->SetTitleSize(0.05);
+  plot->GetXaxis()->SetTitleOffset(0.9);
+  plot->GetYaxis()->SetTitleOffset(1.3);
+  plot->GetXaxis()->SetLabelFont(42);
+  plot->GetYaxis()->SetLabelFont(42);
+  plot->GetYaxis()->SetLabelSize(.05);
+  plot->GetXaxis()->SetLabelSize(.05);
+}
+/*--------------------------------------------------------------------*/
+
+RooRealVar MuMu_mass("MuMu_mass", "MuMu_mass", 70, 110);
+static TString GT = "";
+TLatex* tlxg = new TLatex();
+class FitOut {
+public:
+  double mean;
+  double mean_err;
+  double sigma;
+  double sigma_err;
+  double chi2;
+  FitOut(double a, double b, double c, double d) : mean(a), mean_err(b), sigma(c), sigma_err(d) {}
+};
+
+FitOut ZMassBinFit_OldTool(TH1D* th1d_input, TString s_name = "zmumu_fitting", TString output_path = "./") {

```

Comment:
Why `OldToot`? This naming convention is unclear.

Also, it could be of advantage to write comments à la doxygen to explain what each function is doing.